roles/dalite-create/README.md

```diff
@@ -0,0 +1,4 @@
+Role Name
+=========
+
+Creates an instance for Dalite on OpenStack, probably should be removed. 
```

Comment:
Creating server on OVH automatically might be a good Idea, but lately this didn't work well (I had a lot of problems with failed server provisionings).